tests/e2e/test_rosacli_cluster.go

```diff
@@ -3510,3 +3510,117 @@ var _ = Describe("Sts cluster creation supplemental testing",
 				Expect(zones).To(Equal("[us-east-2a us-east-2b us-east-2c]"))
 			})
 	})
+
+var _ = Describe("Sts cluster with BYO oidc flow creation supplemental testing",
+	labels.Feature.Cluster,
+	func() {
+		defer GinkgoRecover()
+
+		var (
+			rosaClient         *rosacli.Client
+			clusterService     rosacli.ClusterService
+			customProfile      *profilehandler.Profile
+			clusterID          string
+			ocmResourceService rosacli.OCMResourceService
+			testingClusterName string
+		)
+		BeforeEach(func() {
+			By("Init the client")
+			rosaClient = rosacli.NewClient()
+			clusterService = rosaClient.Cluster
+			ocmResourceService = rosaClient.OCMResource
+
+			By("Prepare custom profile")
+			customProfile = &profilehandler.Profile{
+				ClusterConfig: &profilehandler.ClusterConfig{
+					HCP:           false,
+					MultiAZ:       false,
+					STS:           true,
+					OIDCConfig:    "managed",
+					NetworkingSet: false,
+					BYOVPC:        false,
+				},
+				AccountRoleConfig: &profilehandler.AccountRoleConfig{
+					Path:               "/aa/bb/",
+					PermissionBoundary: "",
+				},
+				Version:      "latest",
+				ChannelGroup: "candidate",
+				Region:       "us-east-2",
+			}
+			customProfile.NamePrefix = constants.DefaultNamePrefix
+
+		})
+
+		AfterEach(func() {
+			By("Delete the cluster")
+			if clusterID != "" {
+				rosaClient.Runner.UnsetArgs()
+				_, err := clusterService.DeleteCluster(clusterID, "-y")
+				Expect(err).To(BeNil())
+				rosaClient.Runner.UnsetArgs()
+				err = clusterService.WaitClusterDeleted(clusterID, 3, 35)
+				Expect(err).To(BeNil())
+			}
+			By("Clean resource")
+			errs := profilehandler.DestroyResourceByProfile(customProfile, rosaClient)
+			Expect(len(errs)).To(Equal(0))
+		})
+
+		It("Create STS cluster with oidc config id but no oidc provider via rosacli in auto mode - [id:76093]",
+			labels.Critical, labels.Runtime.Day1Supplemental,
```

Comment:
I noticed that the polarion test case for this says it's a Medium importance test, whichever one isn't accurate should be updated

Reply:
It is a case for cluster creation, I will update the importance in Polarion to Critical.